build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml

```diff
@@ -19,12 +19,7 @@ steps:
     inputs:
       filename: 'set'
 
-  - task: powershell@2
-    inputs:
-      targetType: filePath
-      filePath: build\Install-WindowsSdkISO.ps1
-      arguments: 18323
-    displayName: 'Install Insider SDK (18323)'
+  - template: MUX-InstallWindowsSDK-Steps.yml
```

Comment:
You're a big fan of yml templates. :-)

Reply:
They make the templates way nicer to read and manage (IMO). If you disagree, let me know and I'll try to figure out a different strategy. I only wish there was a way to have one library file with sub-templates you could include. Like functions, yknow?

Reply:
I don't disagree.